packages/paste-core/utilities/text/src/SafelySpreadProps.ts

```diff
@@ -0,0 +1,17 @@
+import {SPACE_PROPS, TYPOGRAPHY_PROPS, OVERFLOW_PROPS} from '@twilio-paste/style-props';
+
+export const PROPS_TO_NULLIFY = {
+  ...SPACE_PROPS,
+  ...TYPOGRAPHY_PROPS,
+  ...OVERFLOW_PROPS,
+  display: undefined,
+  textColor: undefined,
+  verticalAlign: undefined,
+  className: undefined,
+  style: undefined,
+};
+
+export const safelySpreadProps = (props: {}): typeof props & typeof PROPS_TO_NULLIFY => ({
```

Comment:
This is the magic. To safely spread any props onto `Text`, we take `props` and merge with all the `Text` undefined props. This way if you've tried to restyle one of our style props, we reset it back to undefined, before setting the real styles in the component itself.

This isn't typed properly yet, but basically I want to type the return as the type of the props coming in and the `PROPS_TO_NULLIFY` as a combined object. I'm sure you can do it, but I suck at Typescript (seemingly like 99% of the rest of the internet)